.github/actions/install-winget/action.yaml

```diff
@@ -0,0 +1,78 @@
+name: "Install Winget"
+description: "Install winget on windows runners since its not installed by default: https://github.com/actions/runner-images/issues/6472"
+inputs:
+  GITHUB_TOKEN:
+    description: "GitHub token to execute authenticated Github API requests (for higher rate limit)"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Get URIs for Winget v1.11.400 assets
```

Comment:
Is there a reason you increased this? I previously pinned it because certain newer versions were problematic